### ISSUE TITLE
Link to Augment.js on the homepage is going to a male enhancement site

### ISSUE THREAD

**Issue by JoeArizona** [olivernn/lunr.js#218]
Hi all – On the Lunr.js homepage you have a link to Augment.js. However the link goes to a Chinese language male sexual enhancer product site.

The URL in your homepage link to Angular.js: **https://olivernn.github.io/augment.js/**

The URL it redirects to: **http://augmentjs.com/**  (this is the Chinese male enhancer site)

The correct URL for the Augment.js repo on GitHub: **https://github.com/olivernn/augment.js**

Later,

Joe

**Comment by olivernn** [comment]
Ha! That's embarrassing :flushed:, kind of makes sense based on the domain name I guess.

I used to own that domain for the augment.js docs, though it expired a while ago. The docs are hosted using GitHub pages and I forgot to remove the CNAME.

I've [done this now](https://github.com/olivernn/augment.js/commit/909b67cfa89d9cbf178bcd1ca00d73883c79c259) and, for me anyway, it is now showing the right page.

Please re-open if you are still seeing the wrong page, and thanks for reporting!

**Comment by JoeArizona** [comment]
It's been stumping me for a couple of days because the link was still taking me to that site if I used Chrome. I finally tried Edge, Firefox, and Opera, and the link takes me to the correct website in all those browsers, but Chrome was still dishing me to the Chinese site. I couldn't find the cookie in Chrome's cookie manager even though it was setting one. Finally I deleted my Chrome cache (but not my cookies) and it works correctly now. I'm not sure what's in the Chrome cache that would cause a redirect to persist across Chrome restarts and Windows reboots... Interesting glitch. I could easily see the cookies in the other browsers but not in Chrome – maybe it was stored elsewhere, in the cache DB. Thanks for fixing it so quickly.
